src/test/ui/expr/malformed_closure/ruby_style_closure.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
```

Comment:
Let's add a comment with the ticket number and mentioning that this isn't yet handled by the compiler, but we want to at some point.